src/index.ts

```diff
@@ -14,8 +14,19 @@ const getOptionalInput = (name: string) => core.getInput(name) || undefined;
     return;
   }
 
+  let githubUserName = getOptionalInput('githubUserName') || `"github-actions[bot]"`;
+  let githubUserEmail = getOptionalInput('githubUserEmail') || `"github-actions[bot]@users.noreply.github.com"`;
+
   console.log("setting git user");
-  await gitUtils.setupUser();
+ 
+  await gitUtils.setupUser(githubUserName, githubUserEmail);
```

Comment:
nit: we could just group those nicely, like this:
```suggestion
  await gitUtils.setupUser({ name: gitUserName, email: gitUserEmail });
```